### DISCUSSION TITLE
Anyone knows whether this  colour package translate KubelkaMunk into python?

### DISCUSSION THREAD

**Discussion by digital-prepress** [colour-science/colour#969] [Q&A]
https://github.com/colour-science/MunsellAndKubelkaMunkToolbox/tree/master/KubelkaMunk

**Comment by thomasmansencal** [comment]
Hi @digital-prepress,

No sorry, we do not have support for this part of the Munsell and Kubelka-Munk Toolbox.

Cheers,

Thomas
